### PR TITLE
CSHARP-4361: Add log messages to CMAP spec

### PR DESCRIPTION
**Pull request by BorisDog**

### CODE REVIEW

File: tests/MongoDB.Driver.Core.Tests/MongoDB.Driver.Core.Tests.csproj [hunk 1]
```diff
@@ -68,7 +68,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <EmbeddedResource LinkBase="Specifications\connection-monitoring-and-pooling" Include="..\..\specifications\connection-monitoring-and-pooling\**\*.json" />
+    <EmbeddedResource LinkBase="Specifications\connection-monitoring-and-pooling\tests\cmap-format" Include="..\..\specifications\connection-monitoring-and-pooling\tests\cmap-format\*.json" />
```
Comment (DmitryLukyanov) [inline tests/MongoDB.Driver.Core.Tests/MongoDB.Driver.Core.Tests.csproj:71]: nit: why didn't you use `**`?

File: tests/MongoDB.Driver.Tests/Specifications/connection-monitoring-and-pooling/CmapUnifiedTestRunner.cs [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+using MongoDB.Driver.Tests.UnifiedTestOperations;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace MongoDB.Driver.Tests.Specifications.command_logging_and_monitoring
+{
+    public class CmapUnifiedTestRunner : LoggableTestClass
+    {
+        private static readonly HashSet<string> __ignoredLogsMessages = new HashSet<string>(new[]
+            {
```
Comment (DmitryLukyanov) [inline tests/MongoDB.Driver.Tests/Specifications/connection-monitoring-and-pooling/CmapUnifiedTestRunner.cs:30]: I think expected formatting here:

    private static readonly HashSet<string> __ignoredLogsMessages = new HashSet<string>(
         new[]
         {
               ..
         }

**Review by DmitryLukyanov — APPROVED** [review]
Minor comments and then LGTM

File: tests/MongoDB.Driver.Core.Tests/MongoDB.Driver.Core.Tests.csproj (hunk 1 above)
Comment (BorisDog) [reply tests/MongoDB.Driver.Core.Tests/MongoDB.Driver.Core.Tests.csproj:71]: We just need to include the `cmap-format` folder. Logging folder is included in Driver.Tests.

File: tests/MongoDB.Driver.Tests/Specifications/connection-monitoring-and-pooling/CmapUnifiedTestRunner.cs (hunk 2 above)
Comment (BorisDog) [reply tests/MongoDB.Driver.Tests/Specifications/connection-monitoring-and-pooling/CmapUnifiedTestRunner.cs:31]: Done.

**Pull request merged**